STAT/AliDrawStyle.cxx

```diff
@@ -61,11 +61,11 @@
 //
 std::map<TString, TString>  AliDrawStyle::fLatexAlice;
 std::map<TString, TStyle*>  AliDrawStyle::fStyleAlice;
-std::map<TString, std::vector<int>> AliDrawStyle::fMarkerStyles;
-std::map<TString, std::vector<int>> AliDrawStyle::fMarkerColors;
-std::map<TString, std::vector<float>> AliDrawStyle::fMarkerSize;
-std::map<TString, std::vector<int>> AliDrawStyle::fFillColors;
-std::map<TString, std::vector<float>> AliDrawStyle::fLineWidth;
+std::map<TString, std::vector<int> > AliDrawStyle::fMarkerStyles;  // PLEASE LEAVE THE UNAESTHETIC SPACE
```

Comment:
Hallo.

>We will have at some point automatic tests to catch this, we have had some problems integrating them, so no blame here - it's us who have to automatically catch such problems :-)

Today I will continue with modification of class. I would like to ask you how to check my code before pull request.  I expect (maybe I'm wrong) there are not so many cases which we do not catch with default  compiler but are problems for other compilers (ubuntu/SLC/Mac....)
 
Can you provide list of the sed regular expression to indicate, eventually fix errors and aesthetic changes? 
E.g before my commit I removed all space at the end.
 ```
sed 's/ *$//' 
```


Marian.